release/release_tests.yaml

```diff
@@ -1042,7 +1042,7 @@
 
   run:
     timeout: 600
-    script: chmod +x ./run_llama_ft.sh && ./run_llama_ft.sh --size=7b --as-test
+    script: chmod +x ./run_llama_ft.sh && RAY_AIR_NEW_PERSISTENCE_MODE=1 ./run_llama_ft.sh --size=7b --as-test
```

Comment:
do we need this flag in ./run_llama_ft.sh?

Reply:
Yeah, the new codepath isn't on by default at the moment. Going to remove the need for this by 2.7, so I'm not going to update the readme. (this should NOT be merged into the workspace templates branch).